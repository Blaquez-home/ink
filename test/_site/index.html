<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <title>Your New Jekyll Site</title>
    <meta name="viewport" content="width=device-width">
  </head>
  <body class=" ">
    <div class='local-override'>
one <span class='separator'></b> two <span class='separator'></b> three <span class='separator'></b> four five


  title: Another Test Post
title text: Another Test Post
title html: Another Test&nbsp;Post
title url: /2014/02/02/test-post-2.html
linkpost: false

date: 2014-02-02 00:00:00 -0600
date_xml: 2014-02-02T00:00:00-06:00
date_html: <time class='entry-date' datetime='2014-02-02T00:00:00-06:00' pubdate><span class='date-month'>Feb</span> <span class='date-day'>2</span><span class='date-suffix'>nd</span>, <span class='date-year'>2014</span></time>

content: <p>some random post
<a href="/food">relative link</a></p>

  title: Test Post
title text: Test Post →
title html: Test&nbsp;Post&nbsp;<span class='post-marker post-marker-after'>→</span>
title url: http://timecube.com
linkpost: true

date: 2014-02-01 00:00:00 -0600
date_xml: 2014-02-01T00:00:00-06:00
date_html: <time class='entry-date' datetime='2014-02-01T00:00:00-06:00' pubdate><span class='date-month'>Feb</span> <span class='date-day'>1</span><span class='date-suffix'>st</span>, <span class='date-year'>2014</span></time>

content: <p>Test post <strong>strong</strong> <em>emphasized</em> <a href="http://example.com">some link</a></p>



  title: Another Test Post
title text: Another Test Post
title html: Another Test&nbsp;Post
title url: /2014/02/02/test-post-2.html
linkpost: false

date: 2014-02-02 00:00:00 -0600
date_xml: 2014-02-02T00:00:00-06:00
date_html: <time class='entry-date' datetime='2014-02-02T00:00:00-06:00' pubdate><span class='date-month'>Feb</span> <span class='date-day'>2</span><span class='date-suffix'>nd</span>, <span class='date-year'>2014</span></time>

content: <p>some random post
<a href="/food">relative link</a></p>



  title: Test Post
title text: Test Post →
title html: Test&nbsp;Post&nbsp;<span class='post-marker post-marker-after'>→</span>
title url: http://timecube.com
linkpost: true

date: 2014-02-01 00:00:00 -0600
date_xml: 2014-02-01T00:00:00-06:00
date_html: <time class='entry-date' datetime='2014-02-01T00:00:00-06:00' pubdate><span class='date-month'>Feb</span> <span class='date-day'>1</span><span class='date-suffix'>st</span>, <span class='date-year'>2014</span></time>

content: <p>Test post <strong>strong</strong> <em>emphasized</em> <a href="http://example.com">some link</a></p>

</div>

  </body>
</html>
