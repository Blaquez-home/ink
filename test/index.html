---
theme_layout: test
title: Your New Jekyll Site
---
Hello World!

{% embed theme/foo.html greeting='Yo Dawg' %}
