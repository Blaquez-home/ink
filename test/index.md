---
layout: theme:default
title: Your New Jekyll Site
---
**Hello World!**

{% head %}
{% embed theme/foo.html greeting='Yo Dawg' %}
{% endhead %}

{% content_for footer %}
**OMG**
{% endcontent_for %}
