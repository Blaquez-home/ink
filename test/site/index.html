<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <title>Your New Jekyll Site</title>
    <meta name="viewport" content="width=device-width">
    <link href='/stylesheets/all-cefe790baee04da224e27f0e3b192ff3.css' media='all' rel='stylesheet' type='text/css'><link href='/stylesheets/print-8f4e66fc8350adcb4482cebe140f212d.css' media='print' rel='stylesheet' type='text/css'>
    <p>Yo Dawg, I heard you like includes.</p>
  </head>
  <body class="dang">
    <p>Test plugin assets and overrides:</p>

<ul>
<li>layouts</li>
<li>includes</li>
<li>stylesheets</li>
<li>javascripts</li>
<li>files</li>
<li>images</li>
<li>fonts</li>
<li>sass plugin</li>
</ul>

<p>Test configurations</p>

<ul>
<li>sass compression</li>
<li>sass line numbers</li>
<li>concat_css</li>
<li>concat_js</li>
</ul>

<p>Test: markdown
<strong>Hello World!</strong></p>

<p>Test: head tag</p>

<p>Testing booga</p>

    <footer><p><strong>OMG</strong></p></footer>
    <script src='/javascripts/d41d8cd98f00b204e9800998ecf8427e.js'></script>
    
  </body>
</html>
