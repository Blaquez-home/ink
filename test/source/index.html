---
layout: theme:test
title: Your New Jekyll Site
index: true
---

{% assign separator = " <span class='separator'></b> " %}
{% capture foo %}
one
 two
 three

four five

{% endcapture %}{{ foo | join_lines:separator }}
