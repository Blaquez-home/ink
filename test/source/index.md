---
layout: theme:test
title: Your New Jekyll Site
---

Test plugin assets and overrides:

- stylesheets
- javascripts
- files
- images
- fonts
- sass plugin

Test configurations

- sass compression
- sass line numbers
- combine_css
- combine_js

{% assign separator = ' <span class='separator'></b> ' %}
{% capture foo | join_lines:separator %}
one
 two
 three

four five

{% endcapture %}{{ foo }}
