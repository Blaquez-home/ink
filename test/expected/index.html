<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <title>Your New Jekyll Site</title>
    <meta name="viewport" content="width=device-width">
  </head>
  <body class=" ">
    <div class='local-override'><p>Test plugin assets and overrides:</p>

<ul>
<li>layouts</li>
<li>includes</li>
<li>stylesheets</li>
<li>javascripts</li>
<li>files</li>
<li>images</li>
<li>fonts</li>
<li>sass plugin</li>
</ul>

<p>Test configurations</p>

<ul>
<li>sass compression</li>
<li>sass line numbers</li>
<li>concat_css</li>
<li>concat_js</li>
</ul>

<p>Test: markdown
<strong>Hello World!</strong></p>

<p>Test: head tag</p>

<p>Testing Include booga</p>
</div>

  </body>
</html>
